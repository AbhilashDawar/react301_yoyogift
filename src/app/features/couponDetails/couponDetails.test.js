import React from 'react';
import CouponDetails from './presenter.jsx';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

test('categoryView Component', () => {
    const tree = renderer.create(
        <MemoryRouter>
            <CouponDetails />
        </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
