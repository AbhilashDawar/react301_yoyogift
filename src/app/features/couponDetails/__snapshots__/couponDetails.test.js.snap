// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`categoryView Component 1`] = `null`;
